sdk/communication/AzureCommunicationCommon/README.md

```diff
@@ -2,8 +2,8 @@
 
 This package contains common code for Azure Communication Services libraries.
 
-[Source code](https://github.com/Azure/azure-sdk-for-ios/tree/master/sdk/communication/AzureCommunication)
-| [API reference documentation](https://azure.github.io/azure-sdk-for-ios/AzureCommunication/index.html)
+[Source code](https://github.com/Azure/azure-sdk-for-ios/tree/master/sdk/communication/AzureCommunicationCommon)
+| [API reference documentation](https://azure.github.io/azure-sdk-for-ios/AzureCommunicationCommon/index.html)
```

Comment:
I'm not sure how this part of it will work. Do we need to regenerate the docs as well in this push?

Reply:
Yes, I will need to regenerate the docs.